src/pipeline/main.py

```diff
@@ -0,0 +1,84 @@
+from src.utils import *
+
+def create_table():
+    create_tables = [
+        {
+            'database': 'employee_timesheet',
+            'tables': ['raw_employee_data', 'raw_timesheet_data', 'department', 'raw_sales_data', 'employee'],
+            'sql_create': ['../../schema/create_raw_employee_data.sql', '../../schema/create_raw_timesheet_data.sql', '../../schema/create_transform_department_data.sql', '../../schema/create_raw_sales_data.sql', '../../schema/create_transform_employee_data.sql']
+        }
+    ]
+
+    for create_info in create_tables:
+        for index, table in enumerate(create_info['tables']):
+            conn, cursor = db_connection(create_info['database'])
+            create_database_table(conn, cursor, table, create_info['database'], create_info['sql_create'][index])
+            delete_table_records(conn, cursor, table)
```

Comment:
I don't think there will be any records at the time of table creation. So, why are we deleting here?